packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -89,7 +96,13 @@ async function main() {
 
   const governorV2Factory: GovernorV2Ethers__factory = await getContractFactory("GovernorV2");
 
-  const governorV2 = await governorV2Factory.deploy(finder.address, 0);
+  const governorProposals = await governor.queryFilter(governor.filters.NewProposal(null, null));
```

Comment:
This does not account for proposals made on the prior implementation of Governor. We should use `numProposals()` insdead

Reply:
Also we should add 1 to `numProposals()` so to account for the upgrade proposal on the current Governor

Reply:
Thanks @Reinis-FRP you are right